packages/compass-query-bar/src/stores/query-bar-reducer.ts

```diff
@@ -424,6 +424,7 @@ const saveRecentQuery = (
           existingQuery._id,
           updateAttributes
         );
+        _dispatch(fetchSavedQueries());
```

Comment:
Super nit: underscore is a convention for naming function arguments when they are not used, as you are using it now, you should rename to just `dispatch`